lib/cli.js

```diff
@@ -270,6 +271,16 @@ exports.run = async () => { // eslint-disable-line complexity
 
 	const files = cli.input.map(file => path.relative(resolveTestsFrom, path.resolve(process.cwd(), file)));
 
+	for (const file of files) {
+		if (!fs.existsSync(file)) {
+			throw new Error(`No such file: ${cli.input[files.indexOf(file)]}`);
+		}
+
+		if (!fs.lstatSync(file).isFile()) {
```

Comment:
I think this should use `statSync()`, since `lstatSync()` doesn't follow symlinks.